packages/dashboard/app/withdraw/utils/layout.tsx

```
@@ -0,0 +1,37 @@
+'use client';
+
+import React from 'react';
+import { StytchProvider } from '@stytch/nextjs';
+import { createStytchUIClient } from '@stytch/nextjs/ui';
+import { SharedProviders } from '../../layout';
+
+// Create Stytch client only on the client side
+let stytchClient: any = null;
+if (typeof window !== 'undefined') {
+  try {
+    const publicToken = process.env.NEXT_PUBLIC_STYTCH_PUBLIC_TOKEN;
+    if (publicToken) {
+      stytchClient = createStytchUIClient(publicToken);
+    }
+  } catch (error) {
+    console.error('Error initializing Stytch client:', error);
+  }
```

If this fails we definitely don't want to hide the error from the user. It is a great candidate to crash the app as it is basically 66% of our auth methods

---

I can refactor a more comprehensive fix for this in another PR, but it'll require some more time. This issue should only happen when Stytch's service is down since our ENV for it will (should) never change